### PR TITLE
BSD-4 License added to update our previous BSD-3 License

### PR DESCRIPTION
**Pull request by Pop0097**
License text sourced from here: https://choosealicense.com/licenses/bsd-4-clause/

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
-DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
-OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+4. Neither the name of the copyright holder nor the names of its
+   contributors may be used to endorse or promote products derived from
+   this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY COPYRIGHT HOLDER "AS IS" AND ANY EXPRESS OR
```
Comment (Fberbari) [inline LICENSE.md]: "Copyright Holder" should be changed to WARG

File: LICENSE.md [hunk 2]
```diff
@@ -1,29 +1,33 @@
...
-AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
-FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
-OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+4. Neither the name of the copyright holder nor the names of its
```
Comment (Fberbari) [inline LICENSE.md]: copyright holder changed to Warg

File: LICENSE.md [hunk 3]
```diff
@@ -1,29 +1,33 @@
...
+   contributors may be used to endorse or promote products derived from
+   this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY COPYRIGHT HOLDER "AS IS" AND ANY EXPRESS OR
+IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO
+EVENT SHALL COPYRIGHT HOLDER BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
```
Comment (Fberbari) [inline LICENSE.md]: copyright holder to WARG

**Review by Fberbari — CHANGES_REQUESTED** [review]

File: LICENSE.md (hunk 2 above)
Comment (Pop0097) [reply LICENSE.md]: ah I'm dumb: https://spdx.org/licenses/BSD-4-Clause.html

**Review by Fberbari — APPROVED** [review]
sweet

**Comment by Pop0097** [comment]
@Fberbari you do the honours.

**Pull request merged**